docs/authorization/globus_app/apps.rst

```diff
@@ -0,0 +1,45 @@
+.. _globus_apps:
+
+.. currentmodule:: globus_sdk
+
+GlobusApps
+==========
+
+:class:`GlobusApp` is a high level construct designed to simplify the process of
+of authenticating for Globus service clients.
+
+A ``GlobusApp`` requires a programmatic Globus Auth entity called a "client" to be
+supplied as the requester driving authentication flows. A client can be created and
+managed through the `Globus Developer Console <https://developers.globus.org/>`_.
+There are two types of GlobusApps; :class:`UserApp`, for contexts in which a client
+should submit requests on behalf of a user, and :class:`ClientApp`, for contexts which
+a client should submit requests on behalf of themselves.
```

Comment:
Because this is intro text, I'm giving it a bit of extra scrutiny. I think the notion of "clients acting on behalf of themselves" is very abstract and hard for a new user to grasp.

I would phrase these in terms of "login to the app" and "the client acts as an automated user", leading to the following rephrase:
> ```rst
> There are two types of GlobusApp: :class:`UserApp` and :class:`ClientApp`.
> A :class:`UserApp` requires users to login to the application, allowing the application to
> submit requests on behalf of the user.
> A :class:`ClientApp` uses its own credentials to act as a user, and is suitable
> for a variety of automation use cases.
> ```

Reply:
Rephrased it again...

Reply:
FWIW, your rewrite is fantastic!

Marking this thread resolved.

Reply:
Glad to hear you like it! Thanks for pushing me on it.

I'm going to unresolve this thread actually; I pinged Ada & Brigitte about this section to get their sign off as well. Want to leave it open in case they give comments here.